### PR TITLE
Created metadata table with migration and created model

### PR DESCRIPTION
**Pull request by drewpotter**
The basic Metadata model is present along with the database table.

### CODE REVIEW

File: db/migrate/20171204104811_create_metadata.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      t.string :contact_organisation
+      t.string :dataset_id
+      t.string :metadata
+      t.string :factsheet
+      t.boolean :marine_spatial_planning
+      t.boolean :education
+      t.boolean :environmental_impact_assessment
+      t.boolean :ecosystem_assessment
+      t.boolean :ecosystem_services
+
```
Comment (Levia) [inline db/migrate/20171204104811_create_metadata.rb]: Can we please add the timestamps as well?

Comment (drewpotter) [reply db/migrate/20171204104811_create_metadata.rb]: Sorry @Levia I will update this immediately!

**Comment by Levia** [comment]
It looks good. We still need the extra column about the website_download_link though

**Comment by drewpotter** [comment]
Thanks @Levia I just added that.

**Pull request merged**